kak-lsp.toml

```diff
@@ -149,7 +149,12 @@ roots = ["package.json"]
 command = "json-languageserver"
 args = ["--stdio"]
 
-# Requires Julia packages "LanguageServer", "StaticLint" and "SymbolServer"
+# Requires Julia packages "LanguageServer"
+# Run: `julia --project=@kak-lspconfig -e 'import Pkg; Pkg.add("LanguageServer")'` to install it
+# No need to import SymbolServer because it is no longer required
```

Comment:
I think these two lines should be moved to the commit message.
They don't seem useful to new users.

Reply:
sure. i will remove that.